src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
it's bad approach to use @Data in this case, because you have category with fetch type LAZY. When you try to call, for example, toString() generated by Lombok, you'll have an exception. Use only @Getter and @Setter in this case